### ISSUE TITLE
Incorrect assignment of a VCH to a newly created project

### ISSUE THREAD

**Issue by rguske** [goharbor/harbor#7026]
**Expected behavior and actual behavior:**
Creating a new project and assigning a Virtual Container Host (VCH) to this would expect a successful mutual assignment (1:1).
We´ve created a new project, e.g. *vic-poc* and assigned an already deployed VCH to it but the VCH where assigned to the *default-project*.

**Steps to reproduce the problem:**
Deleting the VCH from the *default-project* and reassigning it to *vic-poc* again ends up like before.

**Versions:**
- vic-v1.5.0
```
docker ps -a
goharbor/nginx-photon:v1.7.1
goharbor/notary-server-photon:v0.6.1-v1.7.1
goharbor/harbor-portal:v1.7.1
goharbor/harbor-jobservice:v1.7.1
goharbor/notary-signer-photon:v0.6.1-v1.7.1
goharbor/clair-photon:v2.0.7-v1.7.1
goharbor/harbor-core:v1.7.1
goharbor/harbor-db:v1.7.1
goharbor/redis-photon:v1.7.1
goharbor/harbor-registryctl:v1.7.1
goharbor/harbor-adminserver:v1.7.1
goharbor/registry-photon:v2.6.2-v1.7.1
goharbor/harbor-log:v1.7.1
vmware/admiral:vic_v1.5.0
vmware/vic-machine-server:ova
goharbor/harbor-migrator:v1.7.0
goharbor/harbor-migrator:v1.7.0
goharbor/harbor-migrator:v1.7.0
goharbor/harbor-migrator/harbor-migrator:v1.7.0
```
**Additional context:**
- used `--no-tlsverify`for the VCH deployment
- Credentials for the VCH are based on *Certificates* (server-cert.pem & server-key.pem)
- Adding the VCH to the new project again, without deleting it before from *default-project*, it will apear on both.

**Log files:**

**Comment by tschwaller** [comment]
When you assign the VCH to `vic-poc` it shows up for a very short time (< 0.5 second) and disappears again. Probably the assignment is created in the `default-project` by a UI-bug, but since this is not expected it shows up for this very short time in the `vic-project` until a UI-check realises it is not there and deletes it from that view.

**Comment by lazarin** [comment]
Issue has been fixed with commit 5bb754da602c4510d7d32346f74f51ed435f5790 a while ago and will be included with the next released version.

This is reproducible from the administration tab only and the workaround now is adding the host through the home tab.

**Comment by reasonerjt** [comment]
Closing since this is not an issue in Harbor.
